mycore-acl/src/main/resources/components/acl/config/mycore.properties

```diff
@@ -0,0 +1,35 @@
+##############################################################################
+# CLI                                                                        #
+##############################################################################
+MCR.CLI.Classes.Internal=%MCR.CLI.Classes.External%,org.mycore.mcr.acl.accesskey.cli.MCRAccessKeyCommands
+
+
+##############################################################################
+# Configure Hibernate Mappings                                               #
+##############################################################################
+MCR.Hibernate.Mappings=%MCR.Hibernate.Mappings%,org.mycore.mcr.acl.accesskey.model.MCRAccessKey
+
+
+##############################################################################
+# Additional Events                                                          #
+##############################################################################
+MCR.EventHandler.MCRObject.011.Class=org.mycore.mcr.acl.accesskey.MCRAccessKeyEventHandler
+MCR.EventHandler.MCRDerivate.011.Class=org.mycore.mcr.acl.accesskey.MCRAccessKeyEventHandler
+
+
+##############################################################################
+# Jersey Resources                                                           #
+##############################################################################
+MCR.RestAPI.V2.Resource.Packages=%MCR.Jersey.Resource.Packages%,org.mycore.mcr.acl.accesskey.restapi.v2
+
+
+##############################################################################
+# URIResolver & ContentTransformer                                           #
+##############################################################################
+MCR.URIResolver.ModuleResolver.accesskey=org.mycore.mcr.acl.accesskey.MCRAccessKeyURIResolver
+
+
+##############################################################################
+#API Drafts
+##############################################################################
+MCR.RestApi.Draft.MCRAccessKey=false
```

Comment:
I would enable this per default, because without this the entire module is useless.